### PR TITLE
FrSky R9MM, R9M Lite Pro receivers, add wiring diagram

### PR DESCRIPTION
**Pull request by olliw42**
title says it. See https://discord.com/channels/1005096100572700794/1005096101239603232/1169501454973800559.

### CODE REVIEW

**Comment by jlpoltrack** [comment]
Small feedback from Discord - should it be stated thatOut is SBUS only? Also applies to R9M as receiver.

**Comment by olliw42** [comment]
that's a good idea
should I just add a sentence, or do a list like for the others? I didn't do a list since it takes up so much space, but it would be more canonical.

**Comment by jlpoltrack** [comment]
I think a sentence is fine.  Would also be good to include on the image, would guess that people refer to that before the text.

**Comment by olliw42** [comment]
it would make the image somewhat ugly and it's not on other images either
if a sentence is at risk to not be seen I guess I would rather vote for a table

**Comment by jlpoltrack** [comment]
Works for me.

**Comment by olliw42** [comment]
converted to table
+ added wiring for R9M Lite Pro + few more conversions ot table

**Comment by jlpoltrack** [comment]
Looks good to me - just one small comment.

File: docs/FRSKY_R9.md [hunk 1]
```diff
@@ -151,7 +158,23 @@ The wiring is similar to when used as a receiver, with the exception that the "I
...
-If you want to use ELRS bootloader and install via the Frsky bootloader and OpenTX or EdgeTX, the R9MM is the smallest and lightest receiver available.  Note: The R9MM has a different [wiring diagram](https://www.expresslrs.org/quick-start/receivers/r9/#wiring-up-your-receiver) than the R9MX.
+If you want to use ELRS bootloader and install via the Frsky bootloader and OpenTX or EdgeTX, the R9MM is the smallest and lightest receiver available.
+
+<img src="images/Frsky_R9MM_wiring_01.jpg" width="720px">
+
+| Pin | Use
+| --- | ---
+| Pin 2 VIN | Power 3.5 - 10 V
+| Pin 3 GND | Ground
+| Pin 5 SBUS OUT | Out ((SBus only))
```
[the remark this answers is not in the document]
Comment (olliw42) [reply docs/FRSKY_R9.md]: good catch. THX. done.

**Comment by olliw42** [comment]
lgtm

**Pull request merged**